src/components/Breadcrumbs/index.tsx

```diff
@@ -16,51 +19,61 @@ interface BreadcrumbsProps {
 const Breadcrumbs: React.VFC<BreadcrumbsProps> = ({ breadcrumbs }) => (
   <Container>
     {breadcrumbs.map((bc, index) => (
-      <Fragment
-        breadcrumb={bc}
-        key={`breadCrumb-${bc.text}`}
-        active={breadcrumbs.length - 1 === index}
-      />
+      <Fragment key={`breadCrumb-${bc.text}`}>
+        <BreadcrumbFragment breadcrumb={bc} />
+        {breadcrumbs.length - 1 !== index && (
+          <PaddedIcon data-cy="breadcrumb-chevron" glyph="ChevronRight" />
+        )}
+      </Fragment>
     ))}
   </Container>
 );
 
-interface FragmentProps {
+interface BreadcrumbFragmentProps {
   breadcrumb: Breadcrumb;
-  active: boolean;
 }
-const Fragment: React.VFC<FragmentProps> = ({ breadcrumb, active }) => {
+const BreadcrumbFragment: React.VFC<BreadcrumbFragmentProps> = ({
+  breadcrumb,
+}) => {
   const { text, to, onClick, ...rest } = breadcrumb;
```

Comment:
I think you still have to guard against the `text` being `undefined`. 

<img width="1113" alt="Screen Shot 2022-10-11 at 9 09 12 AM" src="https://user-images.githubusercontent.com/47064971/195099635-c6224191-77dd-43db-9361-502d4338d70f.png">


I think this would work: 
```suggestion
  const { text = "", to, onClick, ...rest } = breadcrumb;
```